### PR TITLE
Added RN for fix MTM-56734

### PR DESCRIPTION
**Pull request by pkepaSaG**

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Root level managed object is not found by querying with the onlyRoots flag
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md:3]:
```suggestion
title: Root level managed object is now reliably found when querying with the onlyRoots flag
```

File: content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-56734
+version: 10.20.266.0
+---
+Fixed an issue in inventory with finding an asset by the onlyRoots flag when the object is removed from the asset hierarchy and becomes the root object again.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md:17]:
```suggestion
In the inventory, when querying with the `onlyRoots` flag, assets could not be found if the object was removed from the asset hierarchy and became the root object again. This issue has now been fixed and root level managed objects are found when querying with the `onlyRoots` flag.
```

File: content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md (hunk 1 above)
Comment (pkepaSaG) [reply content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md]: Changed.

File: content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md (hunk 2 above)
Comment (pkepaSaG) [reply content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md]: Changed.

**Review by martin-sag — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@pkepaSaG Can be merged from my side. Not sure if you need more approval from your team, but Martin has already approved.

**Pull request merged**